applications/web-browser/page.js

```
@@ -0,0 +1,223 @@
+				/* Note: RE ignorecase flag has already been activated */
+				RE_sub.push("&#0*" + char_uppercase.charCodeAt(0) + entityEnd);
+				RE_sub.push(
+					"&#x0*" +
+						char_uppercase.charCodeAt(0).toString(16) +
+						entityEnd
+				);
+			}
+			RE_sub = "(?:" + RE_sub.join("|") + ")";
+			RE_res += charMap[char_lowercase] = RE_sub;
+		}
+		return (ents[string] = RE_res);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)